proto/vtctlservice.proto

```diff
@@ -28,3 +28,13 @@ import "vtctldata.proto";
 service Vtctl {
   rpc ExecuteVtctlCommand (vtctldata.ExecuteVtctlCommandRequest) returns (stream vtctldata.ExecuteVtctlCommandResponse) {};
 }
+
+// Service Vtctld exposes gRPC endpoints for each vt command.
+service Vtctld {
+  // GetKeyspace reads the given keyspace from the topo and returns it.
+  rpc GetKeyspace(vtctldata.GetKeyspaceRequest) returns (vtctldata.Keyspace) {};
+  // GetKeyspaces returns the names of all keyspaces in the topo.
+  rpc GetKeyspaces(vtctldata.GetKeyspacesRequest) returns (vtctldata.GetKeyspacesResponse) {};
+  // ShowAllKeyspaces returns the keyspace struct of each keyspace in the topo.
+  rpc ShowAllKeyspaces(vtctldata.ShowAllKeyspacesRequest) returns (stream vtctldata.Keyspace) {};
```

Comment:
Is there a compelling reason to return a stream vs just a standard reply/response? Streams are much more difficult to consume and reason about. There's a pretty good discussion about the drawbacks here. 
https://github.com/twitchtv/twirp/issues/70#issuecomment-470367807

There are obviously cases for them, especially inside of Vitess, but if we can avoid them in this server definition, I think we'll be better off.

Reply:
This was a super good thread, thank you for sharing that! I think it hit the nail right on the head w.r.t. "is there a compelling reason" with

> people could easily reach for it too early in order to future-proof their APIs, "just in case we need streams later," and walk themselves into an architectural hole which is very difficult to get out of.

I'll switch it to a unary RPC!

In a separate branch, I was exploring making the streaming collect the keyspaces from the topo and send them to the stream in separate goroutines, and the [complexity exploded pretty quickly](https://gist.github.com/ajm188/cee1f237dfe16f68241531f9ebcaec31).